Documentation/README.md

```diff
@@ -0,0 +1,6 @@
+# Rook Quickstart Guides
+
+There are a few different options for running a Rook cluster for your storage needs.  Kubernetes is the recommended way because of the rich orchestration and scheduling that Kubernetes provides via the Rook operator.
+
+1. [Kubernetes](kubernetes.md) (recommended)
+2. [Standalone](standalone.md)
```

Comment:
we should probably have a section for the "using rook guide" here too like we do on the main readme.